.gitops/charts/traffic-court-dev-values.yaml

```diff
@@ -7,14 +7,14 @@ global:
 
 arc-dispute-api:
   image:
-    tag: "1.44.2"
+    tag: "1.44.4"
```

Comment:
For future reference, I might recommend to separate release notes/project tagging from regular JIRA tickets. Makes it easier if ever we need to roll something back.

Reply:
agreed